boards/arm/nucleo_f303re/nucleo_f303re.dts

```diff
@@ -0,0 +1,80 @@
+/*
+ * Copyright (c) 2020 Paul M. Bendixen
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/dts-v1/;
+#include <st/f3/stm32f303Xe.dtsi>
+#include "arduino_r3_connector.dtsi"
+
+/ {
+	model = "STMicroelectronics STM32F303RE-NUCLEO board";
+	compatible = "st,stm32f303re-nucleo", "st,stm32f303";
+
+	chosen {
+		zephyr,console = &usart2;
+		zephyr,shell-uart = &usart2;
+		zephyr,sram = &sram0;
+		zephyr,flash = &flash0;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+		green_led_2: led_2 {
+			gpios = <&gpiob 13 GPIO_INT_ACTIVE_HIGH>;
+			label = "User LD2";
+		};
+	};
+
+	gpio_keys {
+		compatible = "gpio-keys";
+		user_button: button {
+			label = "User";
+			gpios = <&gpioc 13 GPIO_INT_ACTIVE_LOW>;
+		};
+	};
+
+	aliases {
+		led0 = &green_led_2;
+		sw0 = &user_button;
+	};
+};
+
+&i2c1 {
+	status = "okay";
+	clock-frequency = <I2C_BITRATE_FAST>;
+};
+
+&spi2 {
+	status = "okay";
+};
+
+&usart1 {
+	current-speed = <115200>;
+	status = "okay";
+};
+&usart2 {
+	current-speed = <115200>;
+	status = "okay";
+};
+&usart3 {
+	current-speed = <115200>;
+	status = "okay";
+};
```

Comment:
usart2 and uart3 are not enabled in Kconfig

Reply:
No, but `usart2` is set to be the configured output, `UART_2` is also set to default enabled in Kconfig.defconfig.
in `pinmux.c` the pins configured for `#ifdef CONFIG_UART_2` are `USART_2` pins.
Should there be some sort of renaming?

Reply:
You can have a look to other board to see how it is handled.
My request is to have a coherent configuration across dts/pinmux/kconfig..

Reply:
ok looking into this